__tests__/nc-news8.test.js

```diff
@@ -12,7 +12,53 @@ describe("PATCH /api/articles/:article_id", () => {
   test("should return 200 status code if sent an object with correct data", () => {
     const newVote = 1;
     const updateVote = { inc_votes: newVote };
-
-    return request(app).patch("/api/articles/4").send(updateVote);
+    return request(app).patch("/api/articles/1").send(updateVote).expect(201);
```

Comment:
Your description is saying a 200 but your expect is looking for 201. I think this should be 200 :)

Reply:
done